ddtrace/contrib/psycopg/patch.py

```diff
@@ -47,14 +45,15 @@ def unpatch():
     if getattr(psycopg2, "_datadog_patch", False):
         setattr(psycopg2, "_datadog_patch", False)
         psycopg2.connect = _connect
+        _unpatch_extensions(_psycopg2_extensions)
```

Comment:
is this a bug?

Reply:
I thought it was weird it was not used here, but maybe there is a reason for not unpatching?

Reply:
We have this already, so we should be good?

https://github.com/DataDog/dd-trace-py/blob/7f11feb46abb928c69bdd566a0632736ee875119/tests/contrib/psycopg/test_psycopg.py#L55-L79

Reply:
but why wasn't it failing before?

Reply:
hmm ... my guess is that this test is insensitive to the extensions being patched/unpatched